src/__tests__/App.test.tsx

```diff
@@ -54,7 +54,7 @@ describe('App', () => {
 
   test('Should switch between dark mode and light mode on toggle', async () => {
     // set delay to null to prevent test time-out due to useFakeTimers
-    const user = userEvent.setup({ delay: null });
+    const user = userEvent.setup({ advanceTimers: jest.advanceTimersByTime });
```

Comment:
`advanceTimers` is a newish option, that integrates better with fake timers. This makes it possible to remove `delay: null`, and therefore to be closer to how a user actually interacts with the app.

see https://testing-library.com/docs/user-event/options#advancetimers for more info.